### PR TITLE
feat: SQLAlchemy and Sequelize were added to test pipeline

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update the support level of SQLAlchemy and Sequelize to full.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14521
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by wd0517 — APPROVED** [review]

File: develop/dev-guide-third-party-support.md [hunk 1]
```diff
@@ -257,18 +257,18 @@ If you encounter problems when connecting to TiDB using the tools listed in this
          <td>N/A</td>
       </tr>
       <tr>
-         <td><a href="https://github.com/coleifer/peewee/" target="_blank" referrerpolicy="no-referrer-when-downgrade">peewee</a></td>
-         <td>v3.14.10</td>
-         <td>Compatible</td>
+         <td><a href="https://www.sqlalchemy.org/" target="_blank" referrerpolicy="no-referrer-when-downgrade">SQLAlchemy</a></td>
```
Comment (Oreoxmt) [inline develop/dev-guide-third-party-support.md:260]: Please update the support level of SQLAlchemy in dev-guide-choose-driver-or-orm.md:

https://github.com/pingcap/docs/blob/b6b0fe72e4b6cd5a8515efe97a5b0f8a76edcac2/develop/dev-guide-choose-driver-or-orm.md?plain=1#L275-L278

Comment (Icemap) [reply develop/dev-guide-third-party-support.md:260]: Done

File: develop/dev-guide-third-party-tools-compatibility.md [hunk 2]
```diff
@@ -192,6 +192,9 @@ Unsupported features are:
 - The `READ-UNCOMMITTED` and `SERIALIZABLE` [isolation levels](/system-variables.md#transaction_isolation) are not supported.
 - Modification of a column's `AUTO_INCREMENT` attribute is not allowed by default.
 - `FULLTEXT`, `HASH`, and `SPATIAL` indexes are not supported.
+- `sequelize.queryInterface.showIndex(Model.tableName);` is not supported.
+- `sequelize.options.databaseVersion` is not supported.
+- Adding a foreign key reference by using [`queryInterface.addColumn`](https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface#instance-method-addColumn) is not supported.
```
Comment (Oreoxmt) [inline develop/dev-guide-third-party-tools-compatibility.md:197]:
```suggestion
- `sequelize.queryInterface.showIndex(Model.tableName);` is not supported.
- `sequelize.options.databaseVersion` is not supported.
- Adding a foreign key reference using [`queryInterface.addColumn`](https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface#instance-method-addColumn) is not supported.
```

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**